### ISSUE TITLE
Change js/css file doesn't trigger browserSync to reload the page

### ISSUE THREAD

**Issue by hanfeisun** [yeoman/generator-webapp_DEPRECATED#585]
Currently, the livereload property of `browserSync` looks like this:

```
  livereload: {
    options: {
      files: [
        '<%= config.app %>/{,*/}*.html',
        '.tmp/styles/{,*/}*.css',
        '<%= config.app %>/images/{,*/}*',
        '.tmp/scripts/{,*/}*.js'
      ],
      port: 9000,
      server: {
        baseDir: ['.tmp', config.app],
        routes: {
          '/bower_components': './bower_components'
        }
      }
    }
  },
```

When I change and save `app/index.html`, browserSync works well and reload the page. However, when I change and save `app/scripts/main.js` or `app/styles/main.css`.  The `watch:babel` or `watch:styles` tasks are triggered and the compiled `js` or `css` file are generated in `.tmp` folder, but somehow the browser doesn't reload at all...

**Comment by geertbrit** [comment]
Same here. Straight out of box. Exact same problem

**Comment by geertbrit** [comment]
changing to `'.tmp/styles/**.css'` just works for my simplistic case, but not sure how the match differs from `'.tmp/styles/{,*/}*.css'` to be honest

**Comment by eddiemonge** [comment]
Try adding `watchTask: true` to the browserSync grunt config and see if that fixes it

**Comment by masakura** [comment]
It is thought that the cause of this bug is micromatch. browserSync are using micromatch.

jonschlinkert/micromatch#38

**Comment by tconroy** [comment]
Any updates or workarounds for this? I've changed the globbing pattern to `**/*.scss` but it's not live-reloading.

**Comment by rafaortman** [comment]
Till they solve the issue, check gebrits' suggestion. It works just fine.

**Comment by tconroy** [comment]
not working for me.. Inside `styles/` I have multiple sub-folders (`base/`,`modules/`,etc). I import all these sub-folder files into `styles/main.scss`. However when I modify one of the sub-folder files, my browser is not getting live-reloaded.

**Comment by silvenon** [comment]
Sorry guys, I was trying to find out why it doesn't reload, but haven't solved it yet…

**Comment by graywithanA** [comment]
Same problem here except following [gebrits suggestion](https://github.com/yeoman/generator-webapp/issues/585#issuecomment-130064405) and changing `{,*/}*.js` to `**.js` doesn't fix the issue for me.

Has anyone found a different solution?

**Comment by silvenon** [comment]
We'll switch to using BrowserSync directly, like [shakyShane advises](http://www.shakyshane.com/javascript/nodejs/browser-sync/2014/08/24/browser-sync-plus-grunt/).

**Comment by rudenko-maksym** [comment]
I changed all `.tmp/...` paths to `tmp/...` (removed the dot) and know everything works as expected.

``` bash
$> lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.3 LTS
Release:        14.04
Codename:       trusty
$> node --version
v0.10.33
```

**Comment by baraamashaal** [comment]
@rudenko-maksym 
that's works

**Comment by tconroy** [comment]
Thanks @rudenko-maksym :+1:

**Comment by silvenon** [comment]
The fix is released in [v1.1.0](https://github.com/yeoman/generator-webapp/releases/tag/v1.1.0).
